fuzzers/015-clbnffmux/README.md

```diff
@@ -1,42 +1,39 @@
-Purpose:
-Document nFFMUX family of CLB muxes
-
-Algorithm:
-
-Outcome:
-
-
-CLB.SLICE_X0.AFFMUX.B0 30_00
-CLB.SLICE_X0.AFFMUX.B1 30_01
-CLB.SLICE_X0.AFFMUX.B2 30_02
-CLB.SLICE_X0.AFFMUX.B3 30_03
-CLB.SLICE_X0.BFFMUX.B0 30_27
-CLB.SLICE_X0.BFFMUX.B1 30_26
-CLB.SLICE_X0.BFFMUX.B2 30_25
-CLB.SLICE_X0.BFFMUX.B3 30_24
-CLB.SLICE_X0.CFFMUX.B0 30_35
-CLB.SLICE_X0.CFFMUX.B1 30_36
-CLB.SLICE_X0.CFFMUX.B2 30_37
-CLB.SLICE_X0.CFFMUX.B3 30_38
-CLB.SLICE_X0.DFFMUX.B0 30_62
-CLB.SLICE_X0.DFFMUX.B1 30_61
-CLB.SLICE_X0.DFFMUX.B2 30_60
-CLB.SLICE_X0.DFFMUX.B3 30_59
-CLB.SLICE_X1.AFFMUX.B0 31_00
-CLB.SLICE_X1.AFFMUX.B1 31_01
-CLB.SLICE_X1.AFFMUX.B2 31_02
-CLB.SLICE_X1.AFFMUX.B3 30_04
-CLB.SLICE_X1.BFFMUX.B0 31_25
-CLB.SLICE_X1.BFFMUX.B1 31_27
-CLB.SLICE_X1.BFFMUX.B2 31_26
-CLB.SLICE_X1.BFFMUX.B3 31_24
-CLB.SLICE_X1.CFFMUX.B0 31_35
-CLB.SLICE_X1.CFFMUX.B1 31_38
-CLB.SLICE_X1.CFFMUX.B2 31_37
-CLB.SLICE_X1.CFFMUX.B3 31_36
-CLB.SLICE_X1.DFFMUX.B0 30_58
-CLB.SLICE_X1.DFFMUX.B1 31_61
-CLB.SLICE_X1.DFFMUX.B2 31_62
-CLB.SLICE_X1.DFFMUX.B3 31_60
+Purpose:  
```

Comment:
Would you mind formatting them this way?

<pre lang="no-highlight"><code>
# CLBnFFMUX Fuzzer

## Purpose

Document nFFMUX family of CLB muxes  

## Algorithm

## Outcome

```
CLB.SLICE_X0.AFFMUX.B0 30_00
CLB.SLICE_X0.AFFMUX.B0 30_00
CLB.SLICE_X0.AFFMUX.B0 30_00
... more lines ...
```

From manual O6 testing ;

```
X0 AOUTMUX O6  
X0 BOUTMUX O6  
```

</code></pre>

which will then render like this;

> # CLBnFFMUX Fuzzer
> ## Purpose
> 
> Document nFFMUX family of CLB muxes  
> 
> ## Algorithm
> 
> ## Outcome
> 
> ```
> CLB.SLICE_X0.AFFMUX.B0 30_00
> CLB.SLICE_X0.AFFMUX.B0 30_00
> CLB.SLICE_X0.AFFMUX.B0 30_00
> ... more lines ...
> ```
> 
> From manual O6 testing ;
> 
> ```
> X0 AOUTMUX O6  
> X0 BOUTMUX O6  
> ```

Reply:
Sure thing!

Reply:
Done !